smithy-syntax/src/test/resources/software/amazon/smithy/syntax/formatter/fixes-invalid-doc-comments.formatted.smithy

```diff
@@ -23,19 +23,19 @@ use smithy.api#String
 
 /// 12 (keep)
 @deprecated
-/// 13 (change)
+// 13 (change)
 structure Foo {
     /// 14 (keep)
     @length(
         // 15 (change)
         min: 1
     )
-    /// 16 (change)
+    // 16 (change)
     @since("1.x")
-    /// 17 (TODO: change)
+    // 17 (change)
     bar: String
 
-    // 18 (change)
+    /// 18 (TODO: handle this case somehow)
```

Comment:
This looks like a regression.

Reply:
Unfortunately, it is. In my limited knowledge and time, I couldn't figure a way of accomplishing both. The compromise is retaining the right comments, but not converting trailing doc comments attached to nothing.

Reply:
See my comment here: https://github.com/smithy-lang/smithy/pull/2277#issuecomment-2101091519 - this specific regression should be fixed now